packages/react/src/components/form/BaseInput/index.tsx

```diff
@@ -125,14 +125,19 @@ export default function Input(props: InputProps): JSX.Element {
   } = props;
   const [focus, setFocus] = React.useState(false);
 
+  const handleChange = useCallback(
```

Comment:
⚠️ This is a breaking change, we need to ensure that the work done in LLD so far gets updated if necessary.